README_.md

```diff
@@ -0,0 +1,177 @@
+# 1 BladeDISC
+
+BladeDISC is an end-to-end **DynamIc Shape Compiler** project for machine learning
+workloads, which is one of the key components of the Alibaba's [PAI-Blade](https://www.aliyun.com/activity/bigdata/blade).
+BladeDISC provides general, transparent and ease of use performance optimization
+for TensorFlow/PyTorch workloads on GPGPU and CPU backends. The architecture 
+natively supports dynamic shape workloads, with many considerations in the 
+performance of both static and dynamic shape scenarios. It also supports multiple
+and flexible deployment solutions, including both Plugin Mode inside
+TensorFlow/Pytorch runtime, and Standalone Mode for AOT standalone execution.
+The project is based on [MLIR](https://mlir.llvm.org/) and highly related with
+[mlir-hlo](https://github.com/tensorflow/mlir-hlo) project.
+
+Refer to [our website](https://alibaba.github.io/BladeDISC/) for more
+information, including the setup tutorial, developer guide, demo examples and
+documents for developers.
+
+## 1.1 Features and Roadmap
+
+### 1.1.1 Frontend Framework Support Matrix
+
+|           | TensorFlow [1] | Pytorch [2]  |
+|---------- | -------------- | ------------ |
+| Inference |    Yes         |    Yes       |
+|  Training |    Yes [3]     |  Ongoing     |
+
+[1] TensorFlow ver1.12, 1.15, 2.4 & 2.5 are supported and fully verified. For
+other versions some slight works on adaptation might be needed.
+
+[2] Torch 1.6.0 <= version < 1.9.0 has been fully verified.
+
+[3] Although supported, there's much room for imporvement on Op coverage for
+training workloads.
+
+### 1.1.2 Backend Support Matrix
+
+|    | Memory Intensive Part | Compute Intensive Part | End-to-End Usability |
+|----------- | ------------- | ---------------------- | -------------------- |
+| Nvidia GPU |    Yes        |    Yes                 |    Yes               |
+| AMD GPU    |  Ongoing      |  Ongoing               |     No               |
+| Hygon DCU  |    Yes        |    Yes                 |    Yes               |
+|  X86       |    Yes        |  Not open-sourced yet [1]  |     No           |
+
+[1] The compute intensive part of the X86 backend is already supported on the
+internal version. The code decoupling is ongoing and will be open-sourced soon,
+same for the end-to-end usability.
+
+### 1.1.3 Deployment Solutions
+
+* Plugin Mode - BladeDISC works as a pluggin of TensorFlow or PyTorch. Only the
+  supported Ops are clustered and compiled, and the unsupported ones will be
+  executed by the the host runtime. We recommand this mode to most of the users
```

Comment:
Do we need to give a brief definition to the `host runtime` first? `host` may refer to CPU execution for some GPU developers, leading to misunderstanding.

Or just try to use `original runtime`? example:
```
and the remaining ops will not be touched by BladeDISc and left to the original runtime.
```

Reply:
Yes indeed, done